yarn.lock

```diff
@@ -5216,10 +5216,10 @@ typedarray-to-buffer@^3.1.5:
   dependencies:
     is-typedarray "^1.0.0"
 
-typescript@^4.0.5:
-  version "4.0.5"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.0.5.tgz#ae9dddfd1069f1cb5beb3ef3b2170dd7c1332389"
-  integrity sha512-ywmr/VrTVCmNTJ6iV2LwIrfG1P+lv6luD8sUJs+2eI9NLGigaN+nUQc13iHqisq7bra9lnmUSYqbJvegraBOPQ==
+typescript@^3.9.3:
```

Comment:
Any reason for this downgrade?  If the package.json specified it, then I don't need a better reason. :)

Reply:
It looks like the version listed in package.json for the api was 3.9.3. Just made a commit to update it!